shared/logutil/logutil.go

```diff
@@ -23,3 +26,19 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+// CountdownToGenesis logs the time remains until genesis time
+func CountdownToGenesis(genesisTime time.Time, secondsCount int) {
+	ticker := time.NewTicker(time.Duration(secondsCount) * time.Second)
+
+	for {
+		select {
+		case <-time.NewTimer(genesisTime.Sub(roughtime.Now()) + 1 /* adding one to be able to print the last minute */).C:
+			log.Infof("genesis time\n")
+			return
+
+		case <-ticker.C:
+			log.Infof("%02d minutes to genesis!\n", genesisTime.Sub(roughtime.Now()).Round(time.Minute)/time.Minute+1)
```

Comment:
Instead of logging how many minutes remain, you can log the duration as a string. 
https://play.golang.org/p/r2Y5loimo7R

```suggestion
			log.Infof("%s to genesis.", genesisTime.Sub(roughtime.Now())
```
This way you don't have to round or add 1 and you get a more human readable representation of the time remaining. Also you don't need the new line character, especially when the log might be in JSON format.

Reply:
What about this feedback?

Reply:
The same changes I forgot to push, sorry